File: 27-remove-element/remove-element.c
int removeElement(int* nums, int numsSize, int val) {
    int p = 0;
    for (int i = 0; i < numsSize; i++) {
        if (nums[i] != val) {
            nums[p] = nums[i];
            p++;
        }
    }
    return p;
}

File: 27-remove-element/remove-element.js
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 */
var removeElement = function(nums, val) {
    let p = 0;
    for (let i = 0; i < nums.length; i++) {
        if (nums[i] !== val) {
            nums[p] = nums[i];
            p++;
        }
    }
    return p;
};
